As to not conflict with graphics/sdl_image rename the installed binary to
showimage-kde.

--- CMakeLists.txt.orig	2017-03-29 22:13:25 UTC
+++ CMakeLists.txt
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)

 find_package(ECM 1.3.0 REQUIRED NO_MODULE)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
+set(SHOWIMAGE "showimage-kde")

 add_definitions(-DTRANSLATION_DOMAIN="showimage")

@@ -36,9 +37,9 @@ ecm_install_icons(ICONS 32-apps-showimage.png
                   DESTINATION ${ICON_INSTALL_DIR}
                   THEME hicolor)

-add_executable(showimage ${showimage_SRCS})
+add_executable(${SHOWIMAGE} ${showimage_SRCS})

-target_link_libraries(showimage
+target_link_libraries(${SHOWIMAGE}
                       Qt5::Core
                       Qt5::Widgets
                       KF5::ConfigCore
@@ -51,8 +52,8 @@ target_link_libraries(showimage
                       KF5::XmlGui
 )

-install(TARGETS showimage ${INSTALL_TARGETS_DEFAULT_ARGS})
-install(FILES showimage.desktop DESTINATION ${XDG_APPS_INSTALL_DIR})
+install(TARGETS ${SHOWIMAGE} ${INSTALL_TARGETS_DEFAULT_ARGS})
+install(FILES showimage.desktop DESTINATION ${XDG_APPS_INSTALL_DIR} RENAME ${SHOWIMAGE}.desktop)
 #install(FILES showimage.appdata.xml DESTINATION ${KDE_INSTALL_METAINFODIR})

 add_subdirectory(po)
